runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsOverlay.java

```diff
@@ -110,6 +112,28 @@ public Dimension render(Graphics2D graphics)
 			{
 				modelOutlineRenderer.drawOutline(object, (int)config.borderWidth(), color, config.outlineFeather());
 			}
+
+			if (config.highlightClickbox())
+			{
+				Shape clickbox = object.getClickbox();
+				if (clickbox != null)
+				{
+					Color clickBoxColor = ColorUtil.colorWithAlpha(color, color.getAlpha() / 12);
+
+					graphics.setColor(color);
+					graphics.draw(clickbox);
+					graphics.setColor(clickBoxColor);
+					graphics.fill(clickbox);
+					graphics.setStroke(stroke);
```

Comment:
this is wrong. what are you trying to do with this?

Reply:
As far as I can tell, this is how the click boxes for other plugins are being drawn.  This is consistent with the Herbiboar and Agility overlays.  Is there another way to draw this?

Reply:
Neither the ability nor herbiboar overlay have a `setStroke` call, so I don't understand what you mean

Reply:
then, why isn't this just another `renderPolygon` call?

Reply:
Okay, I think I've resolved all the issues now.